watchtower/server/log.go

```diff
@@ -0,0 +1,29 @@
+package server
+
+import (
+	"github.com/btcsuite/btclog"
+	"github.com/lightningnetwork/lnd/build"
+)
+
+// log is a logger that is initialized with no output filters.  This
+// means the package will not perform any logging by default until the caller
+// requests it.
+var log btclog.Logger
```

Comment:
With the new logging format, I think this still needs to register itself as a logger in `log.go` within the main `lnd` directory.

Reply:
was thinking about doing that when this is no longer dead code, but happy to do it now